cli/command_policy_set_actions.go

```diff
@@ -34,6 +35,7 @@ func (c *policyActionFlags) setup(cmd *kingpin.CmdClause) {
 	cmd.Flag("action-command-timeout", "Max time allowed for an action to run in seconds").Default("5m").DurationVar(&c.policySetActionCommandTimeout)
 	cmd.Flag("action-command-mode", "Action command mode").Default("essential").EnumVar(&c.policySetActionCommandMode, "essential", "optional", "async")
 	cmd.Flag("persist-action-script", "Persist action script").BoolVar(&c.policySetPersistActionScript)
+	cmd.Flag("vss", "Enable Volume Shadow Copy Service integration ('true', 'false', 'inherit')").EnumVar(&c.policySetVSS, booleanEnumValues...)
```

Comment:
nit: it may be good to avoid acronyms, how about `enable-volume-shadow-copy`.

Also, should this be an enum:

- `always` - always uses VSS, fails if it can't create VSS snapshot
- `when-available` - uses VSS when available (running as admin), falls back to creating snapshot without VSS when not running as administrator
- `never` - never uses VSS
- `inherit`